### PR TITLE
MegaAntiCheat: Add version v0.1.0

### PR DESCRIPTION
**Pull request by filip2cz**
Closes [#1219](https://github.com/Calinou/scoop-games/issues/1219)

- [X] I have read the [Contributing Guide](https://github.com/Calinou/scoop-games/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by filip2cz** [comment]
> [Your changes do not pass checks.](https://github.com/ScoopInstaller/GithubActions/wiki/Pull-Request-Checks)
> ### MegaAntiCheat
> 
>     * [x]  Description
> 
>     * [x]  License
> 
>     * [x]  Hashes
> 
>     * [ ]  Checkver
> 
>     * [ ]  Autoupdate

It has checkver and autoupdate

**Comment by filip2cz** [comment]
Okay, nevermind. My bad

File: bucket/MegaAntiCheat.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.1.0",
+    "license": "GPL-3.0",
+    "homepage": "https://github.com/MegaAntiCheat/client-backend",
+    "description": "Fans made TF2 anticheat, which makes list of cheaters.",
```
Comment (Calinou) [inline bucket/MegaAntiCheat.json:5]:
```suggestion
    "description": "Fan-made TF2 client-side anticheat which builds a list of cheaters using collected demo data",

```

File: bucket/MegaAntiCheat.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "0.1.0",
+    "license": "GPL-3.0",
+    "homepage": "https://github.com/MegaAntiCheat/client-backend",
+    "description": "Fan-made TF2 client-side anticheat which builds a list of cheaters using collected demo data",
```
Comment (Calinou) [inline bucket/MegaAntiCheat.json:5]:
```suggestion
    "description": "Fan-made TF2 client-side anticheat which builds a list of cheaters using collected demo data",
```

File: bucket/MegaAntiCheat.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "0.1.0",
+    "license": "GPL-3.0",
+    "homepage": "https://github.com/MegaAntiCheat/client-backend",
+    "description": "Fan-made TF2 client-side anticheat which builds a list of cheaters using collected demo data",
+
+    "url": "https://github.com/MegaAntiCheat/client-backend/releases/download/v0.1.0/client_backend.exe#/megaanticheat.exe",
```
Comment (Calinou) [inline bucket/MegaAntiCheat.json:7]:
```suggestion
    "description": "Fan-made TF2 client-side anticheat which builds a list of cheaters using collected demo data",
    "url": "https://github.com/MegaAntiCheat/client-backend/releases/download/v0.1.0/client_backend.exe#/megaanticheat.exe",
```

File: bucket/MegaAntiCheat.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        [
+            "megaanticheat.exe",
+            "MegaAntiCheat"
+        ]
+    ],
+    "post_install": [
+        "echo ''",
+        "echo ''",
+        "echo '-------------------------------------------------------------------------------------------'",
+        "echo 'Setup guide: https://github.com/MegaAntiCheat/client-backend?tab=readme-ov-file#usage-steps'",
```
Comment (Calinou) [inline bucket/MegaAntiCheat.json:23]:
```suggestion
        "echo 'Setup guide: https://github.com/MegaAntiCheat/client-backend#usage-steps'",
```

File: bucket/MegaAntiCheat.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        ]
+    ],
+    "post_install": [
+        "echo ''",
+        "echo ''",
+        "echo '-------------------------------------------------------------------------------------------'",
+        "echo 'Setup guide: https://github.com/MegaAntiCheat/client-backend#usage-steps'",
+        "echo '-------------------------------------------------------------------------------------------'",
+        "echo ''"
+    ]
```
Comment (Calinou) [inline bucket/MegaAntiCheat.json:26]:
```suggestion
    "notes": [
        "Setup guide:",
        "", 
        "    https://github.com/MegaAntiCheat/client-backend#usage-steps",
    ]
```

File: bucket/MegaAntiCheat.json [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    "bin": "megaanticheat.exe",
+    "shortcuts": [
+        [
+            "megaanticheat.exe",
+            "MegaAntiCheat"
+        ]
+    ],
+    "notes": [
+        "Setup guide:",
+        "",
```
Comment (Calinou) [inline bucket/MegaAntiCheat.json:21]:
```suggestion
        "",
```

**Review by Calinou — APPROVED** [review]
Thanks!

File: bucket/megaanticheat.json [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    "shortcuts": [
+        [
+            "megaanticheat.exe",
+            "MegaAntiCheat"
+        ]
+    ],
+    "notes": [
+        "Setup guide:",
+        "",
+        "    https://github.com/MegaAntiCheat/client-backend#usage-steps",
```
Comment (amorphobia) [inline bucket/megaanticheat.json:22]: trailing comma here, cause `ConvertFrom-Json` failing

**Review by amorphobia — COMMENTED** [review]
please fix it

File: bucket/megaanticheat.json (hunk 7 above)
Comment (filip2cz) [reply bucket/megaanticheat.json:22]: https://github.com/Calinou/scoop-games/pull/1223

**Pull request merged**